Readme.md

```diff
@@ -109,6 +109,33 @@ Then a warning will be sent to the error list if ESLint finds a rule violation.
 
 **Note** that when building a solution that utilizes this project from the command line (i.e. with the `dotnet build` or `msbuild` commands) you have to build this project alone first. Otherwise, you'll get "Local gulp not found" errors. Building from Visual Studio doesn't need any special care.
 
+### Browsersync
+
+This helper enables you to see client-side changes in realtime using [Browsersync](https://browsersync.io). It works by spinning up a local server that watches files (according to the configuration) for changes to be able reload/inject static resources with or without reloading the page for every active browser session. Find more information in the [official documentation](https://browsersync.io/docs/options).
+
+Usage:
+```js
+const cssFilesPath = '**/*.css';
+const jsFilesPath = '**/*.js';
+
+const browsersyncOptions = {    
+    files: [
+        cssFilesPath,
+		jsFilesPath
+    ]
+}
+
+gulp.task('browsersyncStart', () => browsersync.browsersyncServe(browsersyncOptions));
+```
+
+Using [proxy mode](https://browsersync.io/docs/options#option-proxy) requires an existing vhost added in the options, which Browsersync will create a proxy for.
+
+If you are not using the proxy mode, the terminal will ask you to add a snippet just before the closing </body> tag, for example:
+```html
+<script id="__bs_script__">//<![CDATA[
+   	document.write("<script async src='http://HOST:3000/browser-sync/browser-sync-client.js?v=2.26.14'><\/script>".replace("HOST", location.hostname));
```

Comment:
COLI nyomán akkor itt a version már nem kell a querystringbe, ugye?

Reply:
Valid, erre nem gondoltam. Cserébe szó szerint ezt írja ki a console azért is másoltam be.

Reply:
Oké, akkor kiveszem!

Reply:
Sorry, már kivettem.

Reply:
Köszi, hogy kivetted! :D